device/RPI3.conf

```diff
@@ -0,0 +1,32 @@
+# https://www.raspberrypi.org/products/raspberry-pi-3-model-b/
+# https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/
+
+export MAKE_ARGS_DEV="
+CROSS_BINUTILS_PREFIX=/usr/local/aarch64-unknown-freebsd12.1/bin
```

Comment:
future proof this config using ${PRODUCT_ABI}
`CROSS_BINUTILS_PREFIX=/usr/local/aarch64-unknown-freebsd${PRODUCT_ABI}/bin`

Reply:
```suggestion
CROSS_BINUTILS_PREFIX=/usr/local/aarch64-unknown-freebsd${SRCABI}/bin
```